### ISSUE TITLE
Add keyboard navigation to advanced filter settings

### ISSUE THREAD

**Issue by wachthekid2** [wachthekid2/Rjean#244]
Advanced filters cannot be accessed via keyboard navigation. Implement support for Tab and Arrow keys.
